### ISSUE TITLE
Make explicit algo numbers in DNSSEC05

### ISSUE THREAD

**Issue by mattias-p** [zonemaster/zonemaster#818]
If you need to know for which algorithm numbers ALGORITHM_OK is emitted you have to work out which algo numbers aren't listed for any other messages. The situation could be much improved by explicitly listing the algo numbers for ALGORITHM_OK.

**Comment by matsduf** [comment]
I the specifications we do not list the parameters that are sent with the message tags. Maybe we should (maybe we should not), but as it is now, this is an implementation question belonging to Zonemaster-Engine. Since I think the request is already fullfilled by the current implementation, I do not move this issue to Zonemaster-Engine:

```
    ALGORITHM_OK => sub {
        __x    # DNSSEC:ALGORITHM_OK
          "The DNSKEY with tag {keytag} uses algorithm number {algorithm}/({description}), which is OK.", @_;
    },
```
(From [source].)

Or is the issue asking for something else? Specific message tags per algorithms?

[source]: https://github.com/zonemaster/zonemaster-engine/blob/f8624169a13d0c6eb17440172d9157df5f9181a7/lib/Zonemaster/Engine/Test/DNSSEC.pm#L446

**Comment by mattias-p** [comment]
This issue about the following wording in DNSSEC05 in the current develop branch:

> If no message has been outputted for the DNSKEY, output ALGORITHM_OK.

Working out which algo numbers give this behavior is a hassle. This is a regression from DNSSEC05 in the master branch.

**Comment by matsduf** [comment]
> Working out which algo numbers give this behavior is a hassle. This is a regression from DNSSEC05 in the master branch.

Now I think I understand. You are asking for the algorithm numbers to be explicitly stated. Fine. It will not change anything, and can be easily fixed.

**Comment by matsduf** [comment]
Will be resolved by #819.

**Comment by matsduf** [comment]
Resolved.
